util/autoconfig.php

```diff
@@ -0,0 +1,12 @@
+<?php

```

Comment:
Is this just an example of the `autoconfig.php` file used in the `autoinstall`?

Reply:
Yes, this is an example-file.

Reply:
Can you please add a couple lines to `util/README` to explain the file's presence? Thanks!

Reply:
I added some explanations at
- `util/README`
- `doc/Install.md`
- `doc/de/Install.md`